sensors/src/test/scala/com/lynbrookrobotics/potassium/sensors/imu/TestXYPosition.scala

```diff
@@ -0,0 +1,88 @@
+package com.lynbrookrobotics.potassium.sensors.imu
+
+import com.lynbrookrobotics.potassium.Signal
+import com.lynbrookrobotics.potassium.sensors.position.xyPosition
+import com.lynbrookrobotics.potassium.units.Point
+import org.scalatest.FunSuite
+import squants.time.Milliseconds
+import squants.motion.{DegreesPerSecond, FeetPerSecond}
+import squants.space.{Degrees, Feet}
+import squants.time.Seconds
+
+class TestXYPosition extends FunSuite{
+  val origin = new Point(Feet(0), Feet(0))
+
+  test("Test moving straight at 2 ft/s for 1 sec results in moving 2 ft forward") {
+    lazy val startTime = System.currentTimeMillis
+    def timePassed = Milliseconds(System.currentTimeMillis - startTime)
```

Comment:
So if I just call `currentValue(Milliseconds(5))` 200 times, that would simulate this signal ticking for 1 second at a period of 5 milliseconds and 200 hertz?